Tests/Container+SwinjectStoryboardSpec.swift

```diff
@@ -26,7 +26,7 @@ class Container_SwinjectStoryboardSpec: QuickSpec {
                 expect(container.description) ==
                     "[\n"
                     + "    { Service: \(controllerType), Storyboard: SwinjectStoryboardTests.AnimalViewController, "
-                    + "Factory: ((Resolver, \(controllerType))) -> \(controllerType), ObjectScope: graph, InitCompleted: Specified }\n"
+                    + "Factory: (Resolver, \(controllerType)) -> \(controllerType), ObjectScope: graph, InitCompleted: Specified }\n"
```

Comment:
It's difference between Swift 3.0 and 3.1. Swift 3.1 fixed the doubly printed tuple: `((a, b))`
https://github.com/Swinject/Swinject/pull/252